tasks/run-matlab-command/v1/matlab.ts

```diff
@@ -0,0 +1,65 @@
+// Copyright 2024 The MathWorks, Inc.
+
+import * as taskLib from "azure-pipelines-task-lib/task";
+import * as fs from "fs";
+import * as path from "path";
+import { v4 as uuidV4 } from "uuid";
+
+export async function runCommand(command: string, platform: string, architecture: string, args?: string) {
+    // write command to script
+    console.log(taskLib.loc("GeneratingScript", command));
+    taskLib.assertAgent("2.115.0");
```

Comment:
Is this something that we have control over or could this break when Azure updates their agents?

Reply:
The argument for assertAgent is a minimum version (which is not obvious) so I think we're ok. We need some of the features required from some minimum agent version. If they don't have it, it will say upgrade to use this extension task.